app/talk/popular-tags.cjsx

```diff
@@ -0,0 +1,35 @@
+React = require 'react'
+PromiseRenderer = require '../components/promise-renderer'
+{makeHTTPRequest} = require 'json-api-client'
+{Link} = require 'react-router'
+
+module?.exports = React.createClass
+  displayName: 'TalkPopularTags'
+
+  propTypes:
+    header: React.PropTypes.object             # Optional header ex: <h1>Tags</h1>
+    section: React.PropTypes.string.isRequired # Talk section
+    params: React.PropTypes.object.isRequired # URL params
+
+  tagsRequest: ->
+    popularTags = talkClient.root + "/tags/popular?section=#{@props.section}&limit=10"
+    makeHTTPRequest 'GET', popularTags, null, talkClient.headers
```

Comment:
Tags are resources, right? I can hit /tags and get resources back. Instead of strings, return those resources.

Couldn't this be queried as something like `/tags?section=zooniverse&sort=uses&limit=10` where "uses" is whatever measure of popularity it's using?

Sorry I'm being a pain about this stuff, but I thought we decided on following JSON API so we wouldn't run into stuff like this.